packages/UI/CONTRIBUTING.md

```diff
@@ -0,0 +1,168 @@
+# Getting started with the Platform Enablement Team
+
+dev-launchers-platform is a mono repo using yarn. All yarn script lines need to be run in the root folder. 
+
+## Installing and quick started
+1. install yarn, below are several ways to install. 
+    - https://classic.yarnpkg.com/en/docs/install#windows-stable
```

Comment:
this is linking to classic yarn, I think it's not very necessary to have the installation process for yarn here.
maybe it fits better in the root of the monorepo above the following https://github.com/dev-launchers/dev-launchers-platform#monorepo-scripts

Reply:
this is the link that should be in the root readme of the monorepo https://v3.yarnpkg.com/getting-started/install